enthought_example/example_mco/tests/test_example_mco.py

```diff
@@ -91,3 +95,57 @@ def test_failure(self):
                     initial_value=5,
                 )
             ]
+
+
+class TestSubprocessWorkflowEvaluator(unittest.TestCase):
+
+    def setUp(self):
+
+        self.evaluator = SubprocessWorkflowEvaluator(
+            workflow=Workflow(),
+            workflow_filepath="test_probe.json"
+        )
+        self.mock_process = mock.Mock()
+        self.mock_process.communicate = mock.Mock(
+            return_value=(b"2", b"1 0")
+        )
+
+    def test___call_subprocess(self):
+
+        # Test simple bash command
+        stdout = self.evaluator._call_subprocess(
+            'uniq', ['Hello', 'World']
```

Comment:
This sort of works by accident because of the way that `subprocess.Popen` is ambiguous about the command being a string or list.  You probably should test against `['uniq']` to be more confident about the advertised API.